app/components/Layout/Footer.jsx

```diff
@@ -34,21 +71,67 @@ class Footer extends React.Component {
        );
     }
 
-    render() {
+    checkNewVersionAvailable(){
+        let request = oneoff();
```

Comment:
Since this should only get called from the light wallet, wrap the whole method in a `if (__ELECTRON__) {}` statement

Reply:
@svk31 If the user was running their own local web build, they could still be using an old version.

Reply:
In that case it seems weird to direct them to download a new version though. I think we should let users running their own custom builds handling updating those builds on their own.